README.md

```diff
@@ -77,6 +81,10 @@ Run setup script
 
     bin/setup
 
+Install phantomjs 1.9.8 to run Javascript unit tests:
+
+    brew install phantomjs198
```

Comment:
Please also mention phontamjs in the https://github.com/fs/rails-base/blob/js-support/doc/README_TEMPLATE.md